### ISSUE TITLE
printLoginInfo incorrect for non standard ports

### ISSUE THREAD

**Issue by WGrobler** [boringproxy/boringproxy#157]
When using boringproxy server with both -print-login & -https-port flags. The printed output is displaying an incorrect admin login link.
This happens because the printLoginInfo in [boringproxy.go:423](https://github.com/boringproxy/boringproxy/blob/b1eb2cec438b260d0ec804c6d8a53b8815692b9f/boringproxy.go#L423) is unaware of httpsPort.

The printLoginInfo need to be updated to include a check for non standard ports and set URL accordingly.

```
func printLoginInfo(token, adminDomain string, httpsPort int) {
	var url string
	if httpsPort != 443 {
		url = fmt.Sprintf("https://%s:%d/login?access_token=%s", adminDomain, httpsPort, token)
	} else {
		url = fmt.Sprintf("https://%s/login?access_token=%s", adminDomain, token)
	}
	log.Println(fmt.Sprintf("Admin login link: %s", url))
	qrterminal.GenerateHalfBlock(url, qrterminal.L, os.Stdout)
}
```

I had to declare the url string variable before the IF statement otherwise the var is not set when called in the log function. In not familiar with Golang, so please update code if incorrect.

**Comment by anderspitman** [comment]
Your code looks good. Pretty sure that's a proper way to handle this in Golang.

I'm not sure how necessary this is in practice. It doesn't seem like it would be common at all for people to want to run the admin domain on a different HTTPS port. I think you'd either be using boringproxy as your reverse proxy, or if you just want it for the tunneling you'd have it behind another reverse proxy that's handling TLS termination for you.

Can you help me understand your use case better?

**Comment by WGrobler** [comment]
For anyone interested, we had a further discussion surrounding this on [indiebits.io](https://forum.indiebits.io/t/installing-own-certificates-non-standard-ports/67/4).

TL;DR: Using non standard ports is included in boringproxy, but its functionality is limited. Please take note that using it might require some additional setup on your end.
